docs/consume-packages/install-use-packages-nuget-cli.md

```diff
@@ -20,6 +20,30 @@ For most commands, the NuGet CLI tool uses the current directory, unless you spe
 
 For a complete list of commands and their arguments, see the [NuGet CLI reference](../reference/nuget-exe-cli-reference.md).
 
+## Support policy
+
+We are fully committed to supporting the most recent version of NuGet.exe.
```

Comment:
We need to rewrite these for docs. They need to be more direct and sincere. This is more blog-speak to me. That means avoiding pronouns where possible like "We are committed".